src/codegen/riscv64/macro-assembler-riscv64.cc

```diff
@@ -1864,6 +1864,21 @@ void TurboAssembler::RoundHelper(FPURegister dst, FPURegister src,
       fmv_s(dst, src);
     }
   }
+  {
+    Label not_NaN;
+    UseScratchRegisterScope temps2(this);
+    Register scratch = temps2.Acquire();
+    if (std::is_same<F, double>::value) {
+      feq_d(scratch, src, src);
```

Comment:
A comment here would be helpful to describe why this sequence is used because it is not obvious. If I understand correctly, the idea is that `feq_d` will set `scratch` to 0 if `src` is a NaN. If `src` is not a NaN, we branch to the label and do nothing, but if it is, `fmin_d` will set `dst` to the canonical NaN.

Reply:
Yes, i will add comment . thx!